.scripts/semaphore.js

```diff
@@ -0,0 +1,103 @@
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// original sources can be found at https://github.com/abrkn/semaphore.js
+;(function(global) {
```

Comment:
I'm unfamiliar with what the semi-colon means at the beginning here. What is it?

Reply:
Nevermind, I looked it up.